### PR TITLE
chore: build as cjs package

### PR DESCRIPTION
**Pull request by cyyynthia**

### CODE REVIEW

**Comment by DavidAnson** [comment]
Ooh, very happy to see this!! I'd love this PR (plus a release) in order to fix https://github.com/igorshubovych/markdownlint-cli/issues/469 without needing to introduce an async code path. It looks like you've done the work already, but if you'd like a second pair of eyes or anything to get this done, I volunteer. Thank you! :)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "smol-toml",
-	"version": "1.1.4",
+	"version": "1.1.5",
```
Comment (DavidAnson) [inline package.json]: At the risk of being a pest, I think changing the primary export from ESM to CJS should be considered a breaking change (i.e., semver major increment vs. patch increment).

Comment (cyyynthia) [reply package.json]: I see why that'd be the case; but the change shouldn't have any effect on downstream users (the package remains compatible with ESM-style imports) so I'm unsure if a semver major is necessary here 🤔

**Comment by cyyynthia** [comment]
After double checking the change I don't think it changes the behavior of the library enough to warrant a semver major.

It does allow for something new as a side effect (that I quite like, actually) and the possibility to import the library as `TOML` without using `* as`. I'll make this a semver minor bump instead of a patch since that's a new feature

File: package.json (hunk 1 above)
Comment (DavidAnson) [reply package.json]: You're right! I hadn't realized existing uses would continue to work, sorry.

**Comment by DavidAnson** [comment]
v1.2.0 works great, thank you!

**Pull request merged**